hooks/blueprint

```diff
@@ -59,8 +62,8 @@ do
         )
       else
         manifests+=(
-          "ocfp/loggregator-agent.yml"
-          "ocfp/metrics.yml"
+#          "ocfp/loggregator-agent.yml" deprecated over log-cache
```

Comment:
If there's nothing to add to the manifests list, then just eliminate this if-else clause.